src/sql/magic_cmd.py

```diff
@@ -65,8 +66,68 @@ def execute(self, line="", cell="", local_ns=None):
 
             args = parser.parse_args(others)
             return inspect.get_columns(name=args.table, schema=args.schema)
+        elif cmd_name == "test":
+            parser = CmdParser()
+
+            parser.add_argument(
+                "-t", "--table", type=str, help="Table name", required=True
+            )
+            parser.add_argument(
+                "-c", "--column", type=str, help="Column name", required=False
+            )
+            parser.add_argument(
+                "-w",
+                "--within",
+                type=str,
+                help="Whether it is within two numbers",
+                required=False,
+            )
+            parser.add_argument(
+                "-sg",
+                "--strictly-greater-than",
+                type=str,
+                help="Greater than a certain number.",
+                required=False,
+            )
+            parser.add_argument(
+                "-sl",
+                "--strictly-less-than",
+                type=str,
+                help="Less than a certain number.",
+                required=False,
+            )
+            parser.add_argument(
+                "-nn",
+                "--no-nulls",
+                help="Returns rows in specified column that are not null.",
+                action="store_false",
+            )
+
+            args = parser.parse_args(others)
+            query = construct_string_query(args)
+            conn = sql.connection.Connection.current.session
+            res = conn.execute(query).fetchall()
+            return res
```

Comment:
I noticed that this might yield to invalid constructs. For example:

```
%sqlcmd test --within 0,10 --strictly-greater-than 2
```

here, the `0` and `2` are conflicting with each other.

So perhaps it can be something like:

```
# by default, this produces:  X >0 && X <10
%sqlcmd test --greater 0 --lower 10

# becomes X>=0 && X<10
%sqlcmd test --greater-or-equal 0 --lower 10
```

now, if the user passes both `--greater` and `--greater-or-equal` we should throw an error